scripts/deployThalesRoyale/thalesRoyale/upgradeRoyale.js

```diff
@@ -44,6 +44,7 @@ async function main() {
 
 	const ThalesRoyale = await ethers.getContractFactory('ThalesRoyale');
 	const implementation = await upgrades.prepareUpgrade(thalesRoyaleAddress, ThalesRoyale);
+	await upgrades.upgradeProxy(thalesRoyaleAddress, ThalesRoyale);
```

Comment:
please delete this line, in the upgrade script we only prepare the upgrade line above, and then in gnosis we execute the upgrade, because ownership.